go/pkg/bertymessenger/notifiee.go

```diff
@@ -2,9 +2,12 @@ package bertymessenger
 
 import (
 	"sync"
+
+	"github.com/gogo/protobuf/proto"
+	"go.uber.org/multierr"
 )
 
-// should use EventStream types, this is a poc of the notifiee system à la ipfs
+// Notifiee should use EventStream types, this is a poc of the notifiee system à la ipfs
```

Comment:
this comment can be removed or replaced by something like "Notifiee system inspired by ipfs", the first part was a TODO that was addressed